improver_tests/wxcode/wxcode/test_WeatherSymbols.py

```diff
@@ -381,6 +381,17 @@ def test_invert_combination_correctly(self):
             result = plugin.invert_condition(node)
             self.assertEqual(result[1], inverse_outputs[i])
 
+    def test_error(self):
+        """Test that the _invert_comparator method raises an error when the condition
+        cannot be inverted."""
+        plugin = WeatherSymbols()
+        possible_inputs = ["==", "!=", "NOT", "XOR"]
```

Comment:
Hmm - but you could have any of these conditions in your dictionary.  (Although the test is valid.)

Reply:
My thinking is that if we add any of these to that dictionary, then this test will force us to update the unit-tests. Codecov does not check that every entry in a dictionary is touched, only that every line of code is touched.